front/app/containers/Admin/projects/project/proposals/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { Box, Title, Text } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+
+import usePhase from 'api/phases/usePhase';
+import useProjectById from 'api/projects/useProjectById';
+
+import ProjectProposalsManager from 'components/admin/PostManager/ProjectProposalsManager';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import NewIdeaButton from '../../components/NewIdeaButton';
+import messages from '../messages';
+
+type TFilterMenu = 'topics' | 'statuses';
+
+const defaultTimelineProjectVisibleFilterMenu = 'statuses';
+const timelineProjectVisibleFilterMenus: TFilterMenu[] = [
+  defaultTimelineProjectVisibleFilterMenu,
+  'topics',
+];
+
+const AdminProjectProposals = () => {
```

Comment:
New route for proposals as to allow this to diverge from ideation with no messy abstractions.